### DISCUSSION TITLE
*SOLVED* Osmand 2.6.5 won't load certain map-data

### DISCUSSION THREAD

**Discussion by Ome-Ko** [osmandapp/OsmAnd#20359] [Maps Related]
Dear Osmandians,

I rely on Osmand 2.5.6 (500Mb) for personal reasons (my navigation-phone can't handle the new 3.5Gb(!) versions). Now I noticed that some maps load and others don't. For instance the map France_hauts-de-france_europe_2.obf loads without any problems, but the adjacent map France_great-east_europe_2.obf will not reveal it's secrets and defaults to the world base map.
On startup Osmand shows a brief message at the bottom of the map "length=31;index=69"

My simple question is "why?". Anybody any cllues?

Thanks in advance.

**Comment by mariush444** [comment]
As I remember there is new obf format startimg from v.3.

**Comment by vshcherb** [comment]
Maps couldn't be bigger 2 GB individually, so I'm not sure what's the reason here. In total there were no limitation for multiple maps 10-20 GB, even 2.6 version should work with 10-20 maps however it's all very subjective to device operating memory

**Reply by Ome-Ko** [reply]
Good one. I removed the French and German maps and now Belgium is showing whereas it did not previously. I think there must be some mapmax involved on this device. :-)

Thanks for helping out!

**Reply by DmitryAlexei** [reply]
You can also try to deactivate maps. Deactivated maps are bypassed to speed up the program in case you have a lot of loaded maps.

**Reply by Ome-Ko** [reply]
Ah yes, that works as well! Activating a few maps results in a crash, so I can figure out exactly where the mapmax-limit is. :-)

Thanks, and keep up the good work. I'm promoting Osmand where I can!
